SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestEntityInteration/EntityInteractionTestActor.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
+#include "EntityInteractionTestActor.generated.h"
+
+UCLASS(Blueprintable)
```

Comment:
Is this used in blueprints?

Reply:
This is convenient to have it to test the compatibility of C++ declaration with blueprint.